lib/Conch/Route.pm

```diff
@@ -106,6 +106,95 @@ __END__
 
 =pod
 
+Unless otherwise specified all routes require authentication.
+
+=head3 C</doc>
+
+=over 4
+
+=item * Does not require authentication.
+
+=back
+
+=head3 C</ping>
+
+Returns the json document C<{ "status" : "ok" }>.
```

Comment:
Is this a schema somewhere?

Reply:
It was. I thought I'd changed it, apparently I missed that one (or lost it in a rebase). It's fixed now to point to the Ping schema.